aiidalab_widgets_base/structures.py

```diff
@@ -448,15 +444,24 @@ def _default_structure(self):
 
 
 class StructureBrowserWidget(ipw.VBox):
-    """Class to query for structures stored in the AiiDA database."""
+    """Class to query for structures stored in the AiiDA database.
+
+    :param title: Title of the widget displayed on a tab in StructureManagerWidget
+    :type title: string
+    :param query_structure_type: A tuple of Data node types that are searched (default: StructureData, CifData)
+    :type query_structure_type: tuple
+    """
 
     structure = Union([Instance(Atoms), Instance(Data)], allow_none=True)
 
-    def __init__(self, title=""):
+    def __init__(self, title="", query_types=None):
         self.title = title
 
         # Structure objects we want to query for.
-        self.query_structure_type = (DataFactory("structure"), DataFactory("cif"))
+        if query_types:
+            self.query_structure_type = query_types
+        else:
+            self.query_structure_type = (DataFactory("structure"), DataFactory("cif"))
```

Comment:
I am correcting myself here since I did it this way initially. But both StructureData and CifData are imported earlier, so we could just reuse them:
```suggestion
            self.query_structure_type = (StructureData, CifData)
```